static-instrumenter/maven-plugin/src/test/java/io/opentelemetry/contrib/staticinstrumenter/plugin/maven/InstrumenterTest.java

```diff
@@ -7,25 +7,24 @@
 
 import static io.opentelemetry.contrib.staticinstrumenter.plugin.maven.JarTestUtil.assertJarContainsFiles;
 
-import java.io.File;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Arrays;
+import java.util.Collections;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.io.TempDir;
 
-class InstrumenterTest {
+class InstrumenterTest extends AbstractTempDirTest {
 
   @Test
-  void shouldInstrument(@TempDir File tempDir) throws IOException {
+  void shouldInstrument() throws IOException {
     // given
     Instrumenter instrumenter =
         new Instrumenter(
             InstrumentationAgent.createFromClasspathAgent(tempDir.toPath()), tempDir.toPath());
     Path testJar = Paths.get(JarTestUtil.getResourcePath("test.jar"));
     // when
-    Path instrumented = instrumenter.instrument(testJar.getFileName(), Arrays.asList(testJar));
+    Path instrumented =
+        instrumenter.instrument(testJar.getFileName(), Collections.singletonList(testJar));
```

Comment:
we typically static import `singletonList` (https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports)